layouts/partials/scripts.html

```diff
@@ -27,7 +29,7 @@
 <script src="{{ "js/generator.gen.js" | absURL }}"></script>
 {{ end }}
 
-{{ if or (eq .Type "company") (eq .Type "categories") }}
+{{ if or (and (eq .Type "company") (.IsSection)) (eq .Type "categories") }}
```

Comment:
Not sure if this is the same, but if you click on a category,e.g. https://deploy-preview-485--datenanfragen.netlify.app/categories/nonprofit/ an error is thrown and the search bar isn't rendered.

```
**Debug information:**

message := TypeError: P is null
type := error
colno := 4457
filename := https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js
lineno := 14
error.message := P is null
error.name := TypeError
error.stack := 117/<@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:4457
117@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:4651
s@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:564
n@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:433
@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:1502
@https://deploy-preview-485--datenanfragen.netlify.app/js/company-list.gen.js:14:1506

error.context := defaultPrevented := false
eventPhase := 2
isTrusted := true
returnValue := true
code_version := 1.0.0
user_agent := Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:82.0) Gecko/20100101 Firefox/82.0
url.href := https://deploy-preview-485--datenanfragen.netlify.app/categories/nonprofit/
url.origin := https://deploy-preview-485--datenanfragen.netlify.app
url.protocol := https:
url.host := deploy-preview-485--datenanfragen.netlify.app
url.hostname := deploy-preview-485--datenanfragen.netlify.app
url.port := url.pathname := /categories/nonprofit/
url.search := url.hash := 
```

Reply:
@mal-tee That's because of:

https://github.com/datenanfragen/website/blob/0dfdd5275e22237c3dbd895a2a79ccdef32076f1/src/company-list.js#L139

It doesn't find the element and we of course cannot access `(null).parentElement`.